### DISCUSSION TITLE
(Feature request) Copy only, not clone

### DISCUSSION THREAD

**Discussion by Baikeryoyo** [TriliumNext/Trilium#7351] [Q&A] [answered]
### Describe feature

It is found that after the document is copied, the document copied in another place is also modified when the document is modified
I want to copy only, and the copied document has no connection to the source document

**Comment by sottey** [comment, marked as answer]
If you choose "Duplicate Subtree" on a note, this will copy the item, not clone it.

Hope that helps!
